### DISCUSSION TITLE
Which is the best Ubuntu version for installing Securityonion.

### DISCUSSION THREAD

**Discussion by kunalhiremath** [Security-Onion-Solutions/securityonion#10073] [Unsupported Versions]
Which Ubuntu version should I install for securityonion 2.3 or later. For securityonion 2.4 do we need to install Rocky Linux 9?

**Comment by cm-ops** [comment]
https://docs.securityonion.net/en/latest/installation.html#installation-on-ubuntu-or-centos - for 2.3

As of now https://docs.securityonion.net/en/2.4/installation.html#installation-on-rocky-linux-or-ubuntu - for 2.4 with this note _Please note that Ubuntu is not supported for manager nodes and will be phased out in the future._

**Reply by kunalhiremath** [reply]
ok thank you.
